blocks/home-hero/home-hero.js

```diff
@@ -38,15 +38,18 @@ export default async function decorate(block) {
       ANALYTICS_LINK_TYPE_ENGAGEMENT,
     );
   });
-  title.insertAdjacentElement('afterend', overlayContainer);
-  const stripe = document.createElement('div');
-  stripe.classList.add('home-hero-stripe');
-  title.insertAdjacentElement('afterend', stripe);
-  const picture = block.querySelector('picture');
-  const imgSrc = getBackgroundImage(picture);
-  picture.remove();
-  const content = block.querySelector('h1').parentNode;
-  content.classList.add('home-hero-content-container');
+  if (title) {
```

Comment:
instead of taking this picture and making it a backgroundImage, you can just use z-index and move the picture back by 1 level like `z-index: -1` for example. that will be simpler than trying to do all this DOM manipulation.

Reply:
@raymond-h-dev yes. your validation is correct. I just caught the other issue (must have slipped through the cracks). Let's fix that as well while you're in here.